holidayextras/brands
# Styling alton payment page

Jira: https://hxshortbreaks.atlassian.net/browse/FP-2267

What does this PR do? (please provide any background)
Maps the relevant alton brand styles to the payment page.

What tests does this PR have?
None

How can this be tested?

1. Npm link FP-2267 brands repo from the-wall FP-2267
2. Head to the alton payment page using this bit: https://www.altontowersholidays.com/?bits%5Bsb_altonseosbp%5D=1
3. Make sure the page works & looks OK on all devices 

Any tech debt?
Nope 

Screenshots / Screencast
<img width="33%" src="https://user-images.githubusercontent.com/29861480/31706805-2326cf76-b3e2-11e7-8fb8-27c5b3486a6f.png" />

![booking-alton-local holidayextraspartners co uk-payment-tag upgrades basketid 03af5ba487ad21f6dd4943879e3f56e0 versionid c8864552 agent ato01 customercode q roomrates 5bhotelcode 5d resspl roomrates 5bcheckindate 5d 2018-06-13 roomrates 5bb](https://user-images.githubusercontent.com/29861480/31706891-6b094544-b3e2-11e7-9af3-a3a63c4486b7.png)

#### What gif best describes how you feel about this work?
![]()


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
